pkg/resources/container.go

```diff
@@ -0,0 +1,250 @@
+//
+// Copyright 2020 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// CS??? removed icp-serviceid-apikey-secret from CommonSecretCheckNames, CommonSecretCheckDirs,
+// CS???   and CommonSecretCheckVolumeMounts
+// Linter doesn't like "Secret" in string var names so use "Zecret"
+
+package resources
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const DefaultImageRegistry = "hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-amd64"
```

Comment:
shouldn't this be `quay.io`

Reply:
Yeah it should, but I left it like it for testing on kube-system for now. so that we can actually login and check the ui and such I figured anyone else who wants to test can change it to quay.io temporarily. 
We can change it now or later. Your call

Reply:
i think for our own testing, we should make sure we always set the registry via the cr
The customer's shouldn't have to and it should always work for them